app/code/Magento/Contact/Test/Unit/Controller/Index/IndexTest.php

```diff
@@ -3,82 +3,102 @@
  * Copyright © Magento, Inc. All rights reserved.
  * See COPYING.txt for license details.
  */
+declare(strict_types=1);
+
 namespace Magento\Contact\Test\Unit\Controller\Index;
 
+use Magento\Contact\Controller\Index\Index;
 use Magento\Contact\Model\ConfigInterface;
+use Magento\Framework\App\Action\Context;
+use Magento\Framework\App\RequestInterface;
+use Magento\Framework\App\ResponseInterface;
 use Magento\Framework\Controller\ResultFactory;
 use Magento\Framework\Controller\ResultInterface;
+use Magento\Framework\TestFramework\Unit\Helper\ObjectManager as ObjectManagerHelper;
+use Magento\Framework\UrlInterface;
+use PHPUnit\Framework\MockObject\MockObject;
+use PHPUnit\Framework\TestCase;
 
-class IndexTest extends \PHPUnit\Framework\TestCase
+/**
+ * Test covers Magento\Contact\Controller\Index\Index
```

Comment:
Please use `@covers` annotation instead
```suggestion
 * @covers \Magento\Contact\Controller\Index\Index
```